web/crux/src/app/image/image.http.controller.ts

```diff
@@ -68,10 +68,10 @@ export default class ImageHttpController {
   @HttpCode(200)
   @ApiOperation({
     description:
-      "Fetch details of an images within a version. `ProductId` refers to the product's ID, `versionId` refers to the version's ID, `imageId` refers to the image's ID. All are required variables.</br></br>Details come in an array, including `name`, `id`, `tag`, `order`, and config details of the image.",
+      "Fetch details of an images within a version. `productId` refers to the product's ID, `versionId` refers to the version's ID, `imageId` refers to the image's ID. All are required parameters.</br></br>An image consists `name`, `id`, `tag`, `order`, and config details of the image.",
```

Comment:
Should be (fixed part bolded):
- Fetch details of an **image** within a version.
- Image details consists of `name`, `id`, `tag`, `order`, and config details of the image.